### PR TITLE
docs: home updates

### PR DESCRIPTION
**Pull request by hippee-lee**
Second try. The first PR actions seem to have gotten disconnected and it stopped building. 

Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
Updated layout for homepage. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: apps/website/.vuepress/theme/components/Home.vue [hunk 1]
```diff
@@ -4,145 +4,185 @@
...
-          <h4 cds-text="heading" cds-layout="p-b:lg">Enterprise Ready</h4>
-          <p class="section-block" cds-text="message">
-            Many Fortune 500 companies rely on us to support demanding, high-performance environments. Use Clarity for
-            your high stakes, data intensive applications.
-          </p>
+    <div cds-layout="grid cols:12 cols@sm:8 p:sm p@sm:none">
+      <!-- hero -->
+      <section cds-layout="grid col:start-1 col@sm:start-3 m-t:xl m-t@sm:xxl p:sm p@sm:none">
+        <section class="section-content whats-new" cds-layout="col:12">
+          <div cds-layout="">
```
Comment (Shijir) [inline apps/website/.vuepress/theme/components/Home.vue:36]: empty `cds-layout`

File: apps/website/.vuepress/theme/global-components/WhatsNew.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
-  <div cds-layout="vertical gap:md m-t:lg">
-    <h3 cds-text="heading center medium" cds-layout="m-b:xl">What's New</h3>
-    <p cds-text="title semibold" cds-layout="m-b:md">
+  <div cds-layout="vertical gap:lg">
+    <h3 cds-text="heading center medium">What's New</h3>
+    <p cds-text="title semibold" cds-layout="">
```
Comment (Shijir) [inline apps/website/.vuepress/theme/global-components/WhatsNew.vue:4]: empty `cds-layout`

File: apps/website/.vuepress/theme/components/Home.vue (hunk 1 above)
Comment (hippee-lee) [reply apps/website/.vuepress/theme/components/Home.vue]: removed.

File: apps/website/.vuepress/theme/global-components/WhatsNew.vue (hunk 2 above)
Comment (hippee-lee) [reply apps/website/.vuepress/theme/global-components/WhatsNew.vue:4]: added a margin-bottom in this one.

**Pull request merged**